packages/headless/src/features/commerce/context/context-state.ts

```diff
@@ -0,0 +1,22 @@
+import {ContextParams} from '../../../api/commerce/commerce-api-params';
+
+export interface CommerceContextState {
+  trackingId: string;
+  language: string;
+  currency: string;
+  // TODO: Does it make sense for the clientId to be part of the context state?
+  clientId: string;
+  context: ContextParams; // TODO: Should this be spread in its parent instead?
```

Comment:
See my comment below, I think we should just have `url` instead. That said, this should be on the PLP controller I think.